product-links.txt

```diff
@@ -9,7 +9,7 @@ https://docs.camunda.io/docs/components/concepts/job-workers/
 https://docs.camunda.io/docs/guides/model-your-first-process
 https://docs.camunda.io/docs/guides/setup-client-connection-credentials
 https://docs.camunda.io/docs/guides/message-correlation
-https://docs.camunda.io/docs/apis-clients/working-with-apis-clients/
+https://docs.camunda.io/docs/apis-tools/working-with-apis-clients/
```

Comment:
blocking: I don't think the URLs in this file should change, because they are for v8.1, which is not changing from apis-clients yet.

Reply:
You are correct here -- will make a note to revisit once live.